notebooks/depth_time_series.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
<p>I would not describe this as a drastic change. The variance in the data is very large, which I interpret as an indication that we need more records to draw further conclusions. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Noted. Indeed, there's a lot of variance and much of this is due to insignificant amount of data. I will remove the word "drastic" and introduce the "high variance" term.</p>

Reply:
Update. I have updated the analysis for this part.